src/System.Net.NameResolution/src/System/Net/NameResolutionPal.Windows.cs

```diff
@@ -504,7 +504,7 @@ private unsafe struct GetAddrInfoExContext
             public static void FreeContext(GetAddrInfoExContext* context)
```

Comment:
While you are on it, could you please also delete the Size static field and just pass sizeof to AllocHGlobal directly?

Reply:
Deleted.